Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -26,6 +26,12 @@ import struct TSCUtility.BuildFlags
 import enum TSCUtility.Diagnostics
 import struct TSCUtility.Triple
 
+extension String {
```

Comment:
can this be a file private extension, and on `AbsolutePath` instead of on `String`? should it also be named something more descriptive than "escaped", eg "backslashesEscaped"